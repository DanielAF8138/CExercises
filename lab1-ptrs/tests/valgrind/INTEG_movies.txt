==3073889== Memcheck, a memory error detector
==3073889== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3073889== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3073889== Command: ../ptrs -m
==3073889== 
Title: The Shawshank Redemption
Year: 1994
Genre: Drama
The Shawshank Redemption [1994] - Drama
==3073889== 
==3073889== HEAP SUMMARY:
==3073889==     in use at exit: 0 bytes in 0 blocks
==3073889==   total heap usage: 5 allocs, 5 frees, 8,299 bytes allocated
==3073889== 
==3073889== All heap blocks were freed -- no leaks are possible
==3073889== 
==3073889== For lists of detected and suppressed errors, rerun with: -s
==3073889== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
