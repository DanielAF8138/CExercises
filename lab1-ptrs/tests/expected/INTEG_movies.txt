Title: The Shawshank Redemption
Year: 1994
Genre: Drama
The Shawshank Redemption [1994] - Drama
